apps/examples.nameguard.io/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@namehash/nameguard-react";
```

Comment:
@FrancoAguzzi Please help me understand why we added this file and this line of code.

Is this actually needed? If so, why? If it is needed, then are we doing something special to cause it to be needed in the examples.nameguard.io app? If we aren't doing anything special but still need this then shouldn't we be adding some documentation in the readme files for the related packages instructing everyone to do this when making use of the package?